CYBRHardeningCheck/Main.ps1

```diff
@@ -346,9 +363,14 @@ Function New-HTMLReportOutput
 	)
 
 	Begin {
-		$reportDateTime = $(Get-Date -Format $g_DateTimePattern)
+		# Parameter moved to main init
+        # $reportDateTime = $(Get-Date -Format $g_DateTimePattern)
+
 		$htmlFileContent = Get-Content $REPORT_TEMPLATE_PATH
-		$exportFileName = "Hardening_HealthCheck_Report_$($reportDateTime.Replace("/","-").Replace(":","-").Replace(" ","_")).html"
```

Comment:
Why did you move it from this location?
it is not really needed in the overall script, only here where we create the report file

Reply:
@AssafMiron All the Init definitions need to be set before the execution